tsconfig.json

```diff
@@ -8,15 +8,15 @@
     "allowJs": true,
     "skipLibCheck": true,
     "esModuleInterop": true,
-    "module": "esnext",
+    "module": "commonjs",
     "resolveJsonModule": true,
-    "isolatedModules": true,
     "allowSyntheticDefaultImports": true,
     "noImplicitReturns": true,
     "strict": true,
     "forceConsistentCasingInFileNames": true,
     "sourceMap": true,
-    "baseUrl": "src"
+    "baseUrl": "src",
```

Comment:
maybe removing `baseUrl` from here and moving it to subproject will do the 'job'